src/plugins/controls/common/types.ts

```diff
@@ -20,7 +20,7 @@ export interface ParentIgnoreSettings {
 }
 
 export type ControlInput = EmbeddableInput & {
-  query?: Query;
+  query?: Query | AggregateQuery;
```

Comment:
It seems like within controls and dashboard aggregate queries can never be used in a meaningful way - so it should be OK to keep the existing type here. Is there a special reason why that's not possible?